README.md

```diff
@@ -26,8 +26,8 @@ Without further ado, here is the Bonusly Employee Un-Handbook.
  * Maximize the conditions for collaboration, creativity, and responsiveness:
    * Purpose - the sense that your job is important and you are having an impact
    * Transparency - easy access to information about your team's goals and performance
-   * Feedback - on your performance so that you can learn and improve
-   * Community - the bond you share with colleagues as your strive toward a common goals
+   * Feedback - own your performance so that you can learn and improve
```

Comment:
"on" is correct. ie. you want feedback on your performance. Can you remove this change?